data/tribes/barbarians.lua

```diff
@@ -280,40 +280,45 @@ tribes:new_tribe {
    },
 
    ship_names = {
-      "Agilaz",
-      "Aslaug",
-      "Baldr",
-      "Bear",
-      "Beowulf",
-      "Boldreth",
-      "Dellingr",
-      "Fulla",
-      "Gersemi",
-      "Hagbard",
-      "Heidrek",
-      "Heimdallr",
-      "Hnoss",
-      "Hrothgar",
-      "Ingeld",
-      "Karl Hundason",
-      "Khantrukh",
-      "Lynx",
-      "Mani",
-      "Odin",
-      "Red Fox",
-      "Saxnot",
-      "Sigmund",
-      "Sigurd",
-      "Snotra",
-      "Thor",
-      "Thron",
-      "Ullr",
-      "Valdar",
-      "Vili",
-      "Volf",
-      "Wild Boar",
-      "Wolverine",
-      "Yrsa",
+      pgettext("shipname", "Agilaz"),
+      pgettext("shipname", "Aslaug"),
+      pgettext("shipname", "Baldr"),
+      pgettext("shipname", "Bear"),
+      pgettext("shipname", "Beowulf"),
+      -- TRANSLATORS: This Barbarian ship is named after an in-game character
+      pgettext("shipname", "Boldreth"),
+      -- TRANSLATORS: This Barbarian ship is named after an in-game character
+      pgettext("shipname", "Chat’Karuth"),
+      pgettext("shipname", "Dellingr"),
+      pgettext("shipname", "Fulla"),
+      pgettext("shipname", "Gersemi"),
+      pgettext("shipname", "Hagbard"),
+      pgettext("shipname", "Heidrek"),
+      pgettext("shipname", "Heimdallr"),
+      pgettext("shipname", "Hnoss"),
+      pgettext("shipname", "Hrothgar"),
+      pgettext("shipname", "Ingeld"),
+      pgettext("shipname", "Karl Hundason"),
+      -- TRANSLATORS: This Barbarian ship is named after an in-game character
+      pgettext("shipname", "Khantrukh"),
+      pgettext("shipname", "Lynx"),
+      pgettext("shipname", "Mani"),
+      pgettext("shipname", "Odin"),
+      pgettext("shipname", "Red Fox"),
+      pgettext("shipname", "Saxnot"),
```

Comment:
here we should have a comment that this is the name of a famous ship used in the Lore of the Barbarian tribe